UI/Source/OutlineViews/mac/OutlineViewModelDataSource.swift

```diff
@@ -10,6 +10,12 @@ public final class OutlineViewModelDataSource: NSObject, NSOutlineViewDataSource
         self.treeController = NestedModelCollectionTreeController(modelCollection: model)
         (self.modelBinders, self.viewBinders) = OutlineViewModelDataSource.extractBinders(columns)
         self.context = context
+
+        guard let outlineColumn = columns.first(where: { $0.isOutlineColumn }) else {
```

Comment:
This is good change, at least until we support mutating columns, can you also remove the warning log this makes redundant in OutlineViewController:103

Reply:
done